audio-capture update (#1581)
saucelabs/sauce-docs: docs/mobile-apps/features/audio-capture.md

## Using Audio Capture on iOS/iPadOS

You will be able to capture audio from iOS 10+ up.

> You will be able to validate the sound your application is making, and able to enable Voiceover and validate the output of it. 

Reviewer: ```suggestion
You will be able to validate the sound your application is making, and  enable Voiceover and validate the output of it. 
```

> You will be able to validate the sound your application is making, and able to enable Voiceover and validate the output of it. 

Reviewer 2: ```suggestion
You will be able to validate the sound your application is making, enable the Voiceover, and validate its output. 
```
